addon/components/frost-fixed-table.js

```diff
@@ -22,14 +22,17 @@ export default Component.extend({
   propTypes: {
     // options
     columns: PropTypes.arrayOf(ColumnPropType),
-    items: ItemsPropType
+    items: ItemsPropType,
```

Comment:
They default to `[]` so not technically, although there won't be much to display without them.